src/ucp/core/ucp_worker.h

```diff
@@ -235,6 +235,8 @@ typedef struct ucp_worker {
     ucs_cpu_set_t                 cpu_mask;        /* Save CPU mask for subsequent calls to ucp_worker_listen */
     unsigned                      ep_config_max;   /* Maximal number of configurations */
     unsigned                      ep_config_count; /* Current number of configurations */
+    unsigned                      ep_print_config_idx; /* Index of the last ep cfg, which
```

Comment:
imp: maybe align them:
```
type1 filed_name;      /* comment */
type2 long_filed_name; /* comment */
```
but I see that some other fileds are not aligned and too long (e.g. `ucp_ep_h mem_type_ep[UCT_MD_MEM_TYPE_LAST]`)

Reply:
the offset would be too big for some other fields then. let it be this way